src/index.ts

```diff
@@ -1899,9 +1899,10 @@ function FlatpickrInstance(
       positionCalendar(positionElement);
     }
 
-    if (self.config.enableTime === true && self.config.noCalendar === true) {
-      if (
-        self.config.allowInput === false &&
+    if(self.config.noCalendar === false) {
+      focusOnDay(undefined, 7);
+    } else if (self.config.enableTime === true && self.config.noCalendar === true) {
+      if ((self.config.allowInput === false || self.config.wrap) &&
```

Comment:
This fixes an issue where if you have `allowInput: true` but also have `noCalendar: true` and `enableTime: true` and are using the `wrap: true` option, when you open the time picker via keyboard, none of the time fields get focus.  So, here I'm saying if we're using `wrap: true` then we should also focus on the "hours" field because presumably the element that currently has focus is the one marked with `data-toggle`.